app/ai/fertilizer/page.tsx

```
@@ -0,0 +1,572 @@
+  const formatInsights = (response: string) => {
+    if (!response || response.trim() === "") {
+      return (
+        <p className="text-green-200/60 italic">No information available</p>
+      );
+    }
+
+    if (response.includes("❌")) {
+      return <p className="text-red-400 font-medium">{response}</p>;
+    }
+
+    const sections = response.split(/(?=\*\*[🌟⚠️💰⚖️🧠])/);
```

Biome (1.9.4)

[error] A character class cannot match a surrogate pair. Add the 'u' unicode flag to match against them.

A surrogate pair forms a single codepoint, but is encoded as a pair of two characters. Without the unicode flag, the regex matches a single surrogate character.
Safe fix: Add unicode u flag to regex


(lint/suspicious/noMisleadingCharacterClass)